api/BluetoothRemoteGATTService.json

```diff
@@ -114,7 +114,7 @@
               "version_added": false
             },
             "edge": {
-              "version_added": false
+              "version_added": "79"
```

Comment:
Chrome above says false, is that right? Can you update all Chromium browsers together?

Reply:
Confirmed this is in Chrome as well, just updated all the other Chromium browsers!